Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandItemsStoreDelete.java

```diff
@@ -32,19 +34,22 @@ public CommandItemsStoreDelete(ODPDBAccess db, HttpServletRequest request, HttpS
 	protected void processBatchItems(Map<String, String> parameters,
 			ODPDBAccess db, CachedDatastoreService ds, CachedEntity character,
 			List<CachedEntity> batchItems) throws UserErrorMessage {
-		// TODO Auto-generated method stub
+		
 		Key characterKey = character.getKey();
 		StringBuilder storeString = new StringBuilder();
 		for(CachedEntity storeItem:batchItems)
 		{
 			if (GameUtils.equals(characterKey, storeItem.getProperty("characterKey"))==false)
 				continue;
 			
+			// We need the actual item to generate the HTML to add.
+			CachedEntity item = db.getEntity((Key)storeItem.getProperty("itemKey"));

```

Comment:
This would be quite a bit more efficient and faster if you did a batch get of all the items you're going to need before entering this loop.